USE-CASES/hazard-annotations.md

```diff
@@ -0,0 +1,135 @@
+## Title: Connected Hazards
+
+### Submitter(s):
+
+Luca Barbato
+
+### Reviewer(s):
+
+<Suggest reviewers>
+
+### Tracker Issue ID:
+
+<please leave blank>
+
+### Category:
+
+"vertical"
+
+### Class:
+
+<please leave blank>
+
+### Status:
+
+<please leave blank>
+
+### Target Users
+
+- device owners
+- device users
+- device manufacturers
+
+
+### Motivation:
+
+Connected devices have an impact on the physical world, more than often interacting with them may have unintended consequences.
+
+The use-case originated within [SIFIS-Home](https://sifis-home.eu) project, where the focus is on trustworthy connected homes:
+The user should be made fully aware of the hazards related to the connected devices operations, be able to declare policies regarding acceptable risks, and have the home system enforce them.
+
+The focus is on the problem of having the information available and how to deliver it to all the possible actors.
```

Comment:
The original point was on "awareness" and scope has been expanded to include policy and enforcement.  However, we can keep the *focus* on awareness.  Suggest the following:
```suggestion
This use case discusses all three aspects (awareness, policies, and enforcement) but the focus is on the foundational problem of awareness: having the information hazards available and delivering it to all the possible actors.
```